transaction-flow.md

```diff
@@ -200,6 +202,23 @@ If a sender has been assigned an `external_id`, this value can be used to find s
   * If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you should send us an empty list of documents: `"documents": [ ]` in the request. All of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
   * If when creating senders or transactions you get the following error in the response: `"errors":{"documents":[{"error":"blank"}]}` it means that KYC requirements are not yet waived for your account. If we already approved your KYC process and so they should be, then please contact us so we can rectify the issue and update your account accordingly.
   * In case you don't do KYC on your site, then you will need to send us documents that we can use to verify the sender's identity, for more details on this please see the [API reference documentation](https://api.transferzero.com/documentation#documents).
+* WTR2 (Wire Transfer Regulations) was put into place to tackle financial crime and fraudulent activities
```

Comment:
This is a bit misleading, and we don't really need to tell what WTR2 is. (Especially as wtr2 rules are less strict than our default, so we are much better in tackling financial come then WTR2) What we need to specify that if you wish validation rules for senders can be much more relaxed and based on wtr2 - but if we enable this relaxation then cashplus is out (as it needs a stricter rules than wtr2)